### ISSUE TITLE
Icon doesn't work well on a dark status bar

### ISSUE THREAD

**Issue by miklselsoe** [olegam/Postboks#8]
An alternate status bar icon for dark mode, would suit the app:
![icon](https://cloud.githubusercontent.com/assets/910031/12086592/e465f29e-b2cb-11e5-88a9-37391dbb1865.png)

**Comment by olegam** [comment]
I agree. I havn't come to this yet, but I got the following snippet from @casperstorm.
``` obj-c
NSDistributedNotificationCenter *notificationCenter = [NSDistributedNotificationCenter defaultCenter];
RACSignal *styleSignal = [[notificationCenter rac_addObserverForName:@"AppleInterfaceThemeChangedNotification" object:nil] takeUntil:[self rac_willDeallocSignal]];
RACSignal *isDarkModeStatusBarSignal = [styleSignal map:^id(id value) {
    NSDictionary *dict = [[NSUserDefaults standardUserDefaults] persistentDomainForName:NSGlobalDomain];
    id style = [dict objectForKey:@"AppleInterfaceStyle"];
    BOOL darkModeOn = ( style && [style isKindOfClass:[NSString class]] && NSOrderedSame == [style caseInsensitiveCompare:@"dark"] );
    return @(darkModeOn);
}];
```

In case anyone wants to make a try implementing this.

**Comment by boserup** [comment]
Fixed in #15
